react-backend/routes/cbauthentication/checkCB.js

```diff
@@ -0,0 +1,61 @@
+const validator = require('validator');
+
+const express = require('express');
+
+const router = express.Router();
+const getCBAlreadyExists = require('../../database/queries/getCBAlreadyExists');
+
+const strongPassword = new RegExp('^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&*])(?=.{8,})');
+
+router.post('/', (req, res, next) => {
+  let body = '';
+  req.on('data', (chunk) => {
+    body += chunk;
+  });
+
+  req.on('end', () => {
+    const data = JSON.parse(body);
+    const org_name = data.formName.split(' ').join('');
+    console.log(data.formName);
+    if (
+      data.formName.length === 0 ||
+      data.formEmail.length === 0 ||
+      data.formGenre.length === 0 ||
+      data.formPswd.length === 0 ||
+      data.formPswdConfirm.length === 0
+    ) {
+      res.send('noinput');
+    } else if (validator.isEmail(data.formEmail) && validator.isAlpha(org_name, ['en-GB'])) {
+      getCBAlreadyExists(data.formName.toLowerCase(), data.formEmail, (error, result) => {
+        if (error) {
+          console.log('error from getCBAlreadyExists ', error);
```

Comment:
Add some meaningful error handling here. even just:
```js
res.status(500).send({error: errorStuff})
```
as if this errors, the browser will just hang with no response as it is at the moment.